src/main/resources/application.properties

```
@@ -0,0 +1 @@
+spring.profiles.active=develop
```

어제 구두로 말했듯이
이부분은 환경변수로 빼도 괜찮을 것 같은 생각이드네요~

```suggestion
spring.profiles.active=${devbie.environment}
```